t/test.py

```diff
@@ -373,6 +373,14 @@ def test_strptime(self):
         dt2 = jdatetime.datetime(1363, 6, 6, 12, 13, 14)
 
         self.assertEqual(True, dt1 == dt2)
+    
+    def test_strptime_bare(self):
+        date_string = "13630606121314"
+        date_format = "%Y%m%d%H%M%S"
+        dt1 = jdatetime.datetime.strptime(date_string, date_format)
+        dt2 = jdatetime.datetime(1363, 6, 6, 12, 13, 14)
+
+        self.assertEqual(True, dt1 == dt2)
```

Comment:
we can use `self.assertTrue` instead of `assertEqual(True, )`

Reply:
I did it.